egov-persister/esign-persister.yml

```
@@ -0,0 +1,45 @@
+      module: ESIGN
+      objecIdJsonPath: $.id
+      tenantIdJsonPath: $.tenantId
+      auditAttributeBasePath: ESignParameter
+      queryMaps:
+        - query: UPDATE  dristi_esign_pdf SET signedFilestoreId =?,lastModifiedBy =?,lastModifiedTime = ? where id =?;
+          basePath: ESignParameter
+          jsonMaps:
+            - jsonPath: $.ESignParameter.signedFileStoreId
+            - jsonPath: $.ESignParameter.auditDetails.lastModifiedBy
+            - jsonPath: $.ESignParameter.auditDetails.lastModifiedTime
+            - jsonPath: $.ESignParameter.id
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)